script.js

```diff
@@ -0,0 +1,44 @@
+const hamburgerBtn = document.getElementById('hamburger-btn');
+const navMenu = document.getElementById('nav-menu');
+const closeBtn = document.getElementById('close-btn');
+const linkColor = document.getElementsByClassName('link-color');
+
+hamburgerBtn.addEventListener('click', () => {
+  hamburgerBtn.style.display = 'none';
+  navMenu.style.display = 'block';
+  navMenu.style.zIndex = '10';
+  navMenu.style.position = 'absolute';
+  navMenu.style.left = '0';
+  navMenu.style.top = '0';
+  navMenu.style.display = 'flex';
+  navMenu.style.flexDirection = 'column';
+  navMenu.style.gap = '20px';
+  navMenu.style.paddingTop = '33px';
+  navMenu.style.paddingLeft = '22px';
+  navMenu.style.background = '#6070ff';
+  navMenu.style.opacity = '0.9';
+  navMenu.style.width = '100%';
+  navMenu.style.height = '600px';
+  closeBtn.style.display = 'block';
+  closeBtn.style.position = 'absolute';
+  closeBtn.style.top = '0';
+  closeBtn.style.right = '30px';
+  closeBtn.style.marginRight = '20px';
+  for (let i = 0; i < linkColor.length; i += 1) {
+    linkColor[i].style.color = 'white';
+    linkColor[i].style.fontSize = '24px';
+  }
+});
+
+for (let i = 0; i < linkColor.length; i += 1) {
+  linkColor[i].addEventListener('click', () => {
+    navMenu.style.display = 'none';
+    hamburgerBtn.style.display = 'flex';
+  });
+}
+
+closeBtn.addEventListener('click', () => {
+  hamburgerBtn.style.display = 'block';
+  navMenu.style.display = 'none';
+  closeBtn.style.display = 'none';
+});
```

Comment:
- [ ] Good effect :clap: but your code is not fully `DRY` because it repeats the same code for setting multiple styles on `navMenu` and `linkColor` in different parts of the code. `DRY` stands for `Don't Repeat Yourself`, a basic principle of software development aimed at reducing the repetition of information. :+1: 

- [ ] A possible way to make it more `DRY` is to define helper functions to set the styles on these elements and then call those functions in the event listeners. check below :point_down: my suggestions :slightly_smiling_face: 

```suggestion
const hamburgerBtn = document.getElementById('hamburger-btn');
const navMenu = document.getElementById('nav-menu');
const closeBtn = document.getElementById('close-btn');
const linkColor = document.getElementsByClassName('link-color');

const setNavMenuStyles = () => {
  navMenu.style.display = 'block';
  navMenu.style.zIndex = '10';
  navMenu.style.position = 'absolute';
  navMenu.style.left = '0';
  navMenu.style.top = '0';
  navMenu.style.display = 'flex';
  navMenu.style.flexDirection = 'column';
  navMenu.style.gap = '20px';
  navMenu.style.paddingTop = '33px';
  navMenu.style.paddingLeft = '22px';
  navMenu.style.background = '#6070ff';
  navMenu.style.opacity = '0.9';
  navMenu.style.width = '100%';
  navMenu.style.height = '600px';
}

const setLinkColorStyles = () => {
  for (let i = 0; i < linkColor.length; i += 1) {
    linkColor[i].style.color = 'white';
    linkColor[i].style.fontSize = '24px';
  }
}

hamburgerBtn.addEventListener('click', () => {
  hamburgerBtn.style.display = 'none';
  setNavMenuStyles();
  setLinkColorStyles();
  closeBtn.style.display = 'block';
  closeBtn.style.position = 'absolute';
  closeBtn.style.top = '0';
  closeBtn.style.right = '30px';
  closeBtn.style.marginRight = '20px';
});

for (let i = 0; i < linkColor.length; i += 1) {
  linkColor[i].addEventListener('click', () => {
    navMenu.style.display = 'none';
    hamburgerBtn.style.display = 'flex';
  });
}

closeBtn.addEventListener('click', () => {
  hamburgerBtn.style.display = 'block';
  navMenu.style.display = 'none';
  closeBtn.style.display = 'none';
});
```

- [ ] With this modification, the code is now drier because the `setNavMenuStyles()` and `setLinkColorStyles()` functions are defined once and then reused in the relevant parts of the code.